pwnlib/dynelf.py

```diff
@@ -704,12 +704,12 @@ def _resolve_symbol_sysv(self, libbase, symb, hshtab, strtab, symtab):
             "A hash table of Elf32_Word objects supports symbol table access", or see:
             https://docs.oracle.com/cd/E19504-01/802-6319/6ia12qkfo/index.html#chapter6-48031
 
-            struct Elf_Hash {
-                uint32_t nbucket;
-                uint32_t nchain;
-                uint32_t bucket[nbucket];
-                uint32_t chain[nchain];
-            }
+            ``struct Elf_Hash { \
+                uint32_t nbucket; \
+                uint32_t nchain; \
+                uint32_t bucket[nbucket]; \
+                uint32_t chain[nchain]; \
+            }``
```

Comment:
See comments for `pwnlib/tubes/ssh.py`.

I also don't know much about sphinx docstrings formatting, but this should be leaved multi-line IMO so that it still looks multiline in formatted docs.

Reply:
Changed as requested.